ryan-deblois.txt

```diff
@@ -0,0 +1,45 @@
+
+1. What is Semantic HTML? Semantic HTML is the use of markup so that websites can read and present what is on the page correctly with meaning.
+    2. What is HTML used for? HTML is used to have a universal language for other to read and understand as well as computers to properly display what is
+	being asked. 
+    3. What is an attribute and where do we put it? An attribute is a value you give something in HTML such as color, font, alignment, etc.
+    4. What is the h1 tag used for? How many times should I use it on a page? h1 tag is used for the main heading on a page. It should only be used once.
+    5. Name two tags that have required attributes 
+    6. What do we put in the head of our HTML document? <header>
+    7. What is an id? a more specific tag for an element
+    8. What elements can I add an id to? Any element
+    9. How many times can I use the same id on a page? Only once.
+    10. What is a class? Another way to style an element, but can be multiple to the same class.
+    11. What elements can I add a class to? Any element.
+    12. How many times can I use the same class on a page? As many as needed.
+    13. How do I get my link to open in a new tab? Right click choose open in a new tab?
+    14. What is the alt attribute used for? If an image doesn't show the alt image gives text on what it is
+    15. How do I reference an id? #
+    16. What is the difference between a section and a div? a div doesn't have any meaning aside from classes/ids/etc and sections have content that sticks together ona page, usually follows a theme.
+    17. What is CSS used for? Styling the web page
+    18. How to we select an element? Example - every h2 on the page:  h2 {}
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one: class can have multiple elements tied to it id can only have 1. classes could be used for multiple cities and id for a picture.
+    20. How do we select classes in CSS? .className
+    21. How do we select a p element with a single class of “human””? .human
+    22. What is a parent child selector? When would this be useful? anything child to the parent can be selected with 1 use of the specified parent. Multiple links in a <nav> to keep them together
+    23. How do you select all links within a div with the class of sidebar? .sidebar
+    24. What is a pseudo selector? a selector for a specific state such as :hover
+    25. What do we use the change the spacing between lines? <p> or <br>
+    26. What do we use to change the spacing between letters? pixels or em
```

Comment:
what is the selector you would use?

Reply:
letter-spacing is what you would use with a value after i.e. normal, px, or em.